allure-report-face/src/styles.css

```diff
@@ -10,11 +11,13 @@
 @import "./blocks/link/styles.css";
 @import "./blocks/spinner/styles.css";
 @import "./blocks/text/styles.css";
+@import "./blocks/widget/styles.css";
 @import "./variables.css";
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,700&subset=latin,cyrillic);
```

Comment:
try to compare system font and open sans. And you'll see the difference. Use your head before such advices plz :)